ui/src/plugins/dev.perfetto.StandardGroups/index.ts

```diff
@@ -45,6 +46,7 @@ export default class implements PerfettoPlugin {
     NETWORK: makeGroupNode('Network'),
     DEVICE_STATE: makeGroupNode('Device State'),
     SYSTEM: makeGroupNode('System'),
+    KERNEL: makeGroupNode('Kernel'),
```

Comment:
I would actually call this LINUX to distinguish from other kernels we now support (e.g. QNX).